styleguide/derek/examples/index.jade

```diff
@@ -69,4 +69,13 @@
       a(href="derek/examples/events-interior")
         img.img-thumbnail(src="imgs/derek/examples/event-interior.png")
     .col-md-4
+      h5 Story Overview
+        a(href="derek/examples/stories")
+          img.img-thumbnail(src="imgs/derek/examples/story-overview.png")
+  .row
+    .col-md-4
+      h5 Story Interior
+        a(href="derek/examples/stories-interior")
+          img.img-thumbnail(src="imgs/derek/examples/story-interior.png")
+    .col-md-4
```

Comment:
In order to slim this stuff down, can we assume if they get to the overview page of events or stories that they can get to the interior and remove those from the examples page?